src/application/components/CacheSize.tsx

```diff
@@ -0,0 +1,305 @@
+      <p>
+        It is called with transformed <code>DocumentNode</code>s.
+      </p>
+    </>
+  ),
+  "documentTransform.cache": (
+    <>
+      <p>
+        Cache size for the cache of
+        <TooltipLink href="https://github.com/apollographql/apollo-client/blob/main/src/utilities/graphql/DocumentTransform.ts">
+          <code>DocumentTransform</code>
+        </TooltipLink>{" "}
+        instances with the <code>cache</code> option set to <code>true</code>.
+      </p>
+      <p>
+        Can be called with user-defined or already-transformed{" "}
+        <code>DocumentNode</code>s.
+      </p>
+    </>
+  ),
+  "fragmentRegistry.lookup": (
+    <>
+      <p>
+        A cache inside of
+        <TooltipLink href="https://github.com/apollographql/apollo-client/blob/main/src/cache/inmemory/fragmentRegistry.ts">
+          <code>FragmentRegistry</code>
+        </TooltipLink>
+        .
+      </p>
+      <p>
+        This function is called with fragment names in the form of a string.
+      </p>
+    </>
+  ),
+  "fragmentRegistry.findFragmentSpreads": (
+    <>
+      <p>
+        Cache size for the <code>findFragmentSpreads</code> method of
+        <TooltipLink href="https://github.com/apollographql/apollo-client/blob/main/src/cache/inmemory/fragmentRegistry.ts">
+          <code>FragmentRegistry</code>
+        </TooltipLink>
+        .
+      </p>
+      <p>
+        This function is called with transformed <code>DocumentNode</code>s, as
+        well as recursively with every fragment spread referenced within that,
+        or a fragment referenced by a fragment spread.
+      </p>
+    </>
+  ),
+  "fragmentRegistry.transform": (
+    <>
+      <p>
+        A cache inside of
+        <TooltipLink href="https://github.com/apollographql/apollo-client/blob/main/src/cache/inmemory/fragmentRegistry.ts">
+          <code>FragmentRegistry</code>
+        </TooltipLink>
+        .
+      </p>
+      <p>
+        Can be called with user-defined or already-transformed{" "}
+        <code>DocumentNode</code>s.
+      </p>
+    </>
+  ),
+  "cache.fragmentQueryDocuments": (
+    <>
+      <p>
+        Cache size for the <code>getFragmentDoc</code> method of
+        <TooltipLink href="https://github.com/apollographql/apollo-client/blob/main/src/cache/core/cache.ts">
+          <code>ApolloCache</code>
+        </TooltipLink>
+        .
+      </p>
+      <p>This function is called with user-provided fragment definitions.</p>
+    </>
+  ),
+  "inMemoryCache.executeSelectionSet": (
+    <>
+      <p>
+        Cache size for the <code>executeSelectionSet</code> method on{" "}
+        <TooltipLink href="https://github.com/apollographql/apollo-client/blob/main/src/cache/inmemory/readFromStore.ts">
+          <code>StoreReader</code>
+        </TooltipLink>
+        .
+      </p>
+      <p>
+        Note: <code>executeSelectionSet</code> will be set to the{" "}
+        <code>resultCacheMaxSize</code> option and will fall back to this
```

Comment:
We removed `resultCacheMaxSize` in 4.0, so this note should be removed or displayed conditionally (here and in 2 more places).

Reply:
Seems we have an out-of-date doc-block in Apollo Client itself since I got this straight from there. I'll open a PR over there to fix. Eventually it would be nice to have this come straight from the doc block itself.